### ISSUE TITLE
image classification using keras

### ISSUE THREAD

**Issue by dongniue** [fchollet/keras#4866]
Below are some of my codes.

train_data_dir = 'forTrainTest/Train'
validation_data_dir = 'forTrainTest/Test'

image_width, image_height = 160, 90

model = Sequential()

model.add(Convolution2D(16, 3, 3, border_mode = 'same', input_shape=(image_width, image_height, 3)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2), border_mode = 'same'))

model.add(Convolution2D(32, 3, 3, border_mode = 'same'))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2), border_mode = 'same'))

model.add(Flatten())
model.add(Dense(128))
model.add(Activation('relu'))

model.add(Dense(4))
model.add(Activation('softmax'))

model.compile(loss='categorical_crossentropy',
optimizer='rmsprop',
metrics=['accuracy'])

this is the augmentation configuration we will use for training

train_datagen = ImageDataGenerator(
rescale=1./255,
rotation_range=0,
shear_range=0.0,
zoom_range=0.0,
horizontal_flip=False)

this is the augmentation configuration we will use for testing:

only rescaling

test_datagen = ImageDataGenerator(rescale=1./255)

train_generator = train_datagen.flow_from_directory(
train_data_dir,
target_size=(image_width, image_height),
batch_size=50,
class_mode='categorical')

validation_generator = test_datagen.flow_from_directory(
validation_data_dir,
target_size=(image_width, image_height),
batch_size=50,
class_mode='categorical')

model.fit_generator(
train_generator,
samples_per_epoch=9000*4,
nb_epoch=1,
validation_data=validation_generator,
nb_val_samples=11175)

I noticed that for input_shape of conv_1, it should be 3*width*height, I tried but it gives an error all the time, now I am using width*height*3, no error but it seems not the correct way...

Also I am not sure about overall structure of the codes, but the training accuracy is way too high (98%), and validation seems to be 1.0 (100%). I think there might be some error in the codes but I have no idea what it is...

Thanks in advance, I really wanna make sure my overall codes structure is correct before playing with the net layers...
